projects/plugins/jetpack/_inc/client/recommendations/feature-summary/primary.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { getSummaryPrimaryProps } from '../feature-utils';
+
+const SummaryTextLink = ( { href, label } ) => (
+	<a rel="noreferrer" target="_blank" href={ href } className="jp-summary-text-link">
+		{ label }
+		<span class="jp-summary-text-link__icon dashicons dashicons-arrow-right-alt2"></span>
+	</a>
+);
```

Comment:
Another comment on this, would it be better for these links to look the same as the other buttons on the summary page? Like this:
![image](https://user-images.githubusercontent.com/65001528/193331640-6a228cb3-3f3d-4516-9634-f32f46621251.png)
instead of this:
![image](https://user-images.githubusercontent.com/65001528/193331687-a5a3b157-a24a-4360-868b-0f8747ca9da0.png)

Or were these designed to be vastly different as to separate them visually from the rest?

Reply:
They have been designed that way on purpose, as I understand to appear less actionable than the recommendations CTAs. I will ask the design team for confirmation.

Reply:
I figured that would be a possibility, was just making sure 😄